DB/projects.json

```diff
@@ -77,13 +78,51 @@
       {
         "link": "https://github.com/Sahilll15/Travel-website-css",
         "title": "TravelWebsite",
-        "description": "An Basic Travel webiste",
+        "description": "TravelWebsite is a basic travel website designed with HTML, CSS, and JavaScript. Experience the allure of various travel destinations showcased in this simple yet visually appealing website.",
         "tech": [
           "html",
           "css",
           "javascript"
         ]
       }
+      ,
+      {
+        "link": "https://github.com/Sahilll15/Anonymous_Social_frontend",
+        "title": "Anonymous_Social",
+        "description": "Anonymous_Social is a unique social media platform where users can post comments and manage their profiles anonymously. Built using Node.js, ReactJS, ExpressJS, Tailwind CSS, and MongoDB, this platform ensures user privacy while fostering engaging interactions.",
+        "tech": [
+          "nodejs",
+          "reactjs",
+          "expressjs",
+          "tailwindcss",
+          "mongodb"
+
+        ]
+      },
+      {
+        "link": "https://github.com/Sahilll15/Anonymous_Social_frontend",
```

Comment:
you have added the same link
line no `90`
please check

Thank you for changing description. So try to create separate issues & PRs in future for different changes it will help us to maintain the project

Reply:
okay sorry for that!!